CRITICAL - 2024-11-27 00:50:29 --> ParseError: syntax error, unexpected double-quote mark
[Method: GET, Route: create]
in APPPATH\Views\componentes\head.php on line 4.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\View\View.php(487): CodeIgniter\View\View->render('componentes/head.php', null, true)
 3 APPPATH\Views\eventos\create.php(5): CodeIgniter\View\View->include('componentes/head.php')
 4 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\proyectofinalweb\\app\\Views\\eventos\\create.php')
 5 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('eventos/create', [], true)
 7 APPPATH\Controllers\EventoController.php(27): view('eventos/create')
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EventoController->create()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EventoController))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-27 01:32:04 --> ParseError: syntax error, unexpected token "array"
[Method: GET, Route: create]
in APPPATH\Controllers\EventoController.php on line 31.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\laragon\\www\\proyectofinalweb\\app\\Controllers\\EventoController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\EventoController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\EventoController')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\EventoController', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-27 01:32:08 --> ParseError: syntax error, unexpected token "array"
[Method: GET, Route: /]
in APPPATH\Controllers\EventoController.php on line 31.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\laragon\\www\\proyectofinalweb\\app\\Controllers\\EventoController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\EventoController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\EventoController')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\EventoController', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-27 01:32:13 --> ParseError: syntax error, unexpected token "array"
[Method: GET, Route: /]
in APPPATH\Controllers\EventoController.php on line 31.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\laragon\\www\\proyectofinalweb\\app\\Controllers\\EventoController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\EventoController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\EventoController')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\EventoController', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-27 01:32:26 --> ParseError: syntax error, unexpected token "}", expecting ";"
[Method: GET, Route: /]
in APPPATH\Controllers\EventoController.php on line 41.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\laragon\\www\\proyectofinalweb\\app\\Controllers\\EventoController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\EventoController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\EventoController')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\EventoController', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-27 01:36:24 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\laragon\www\proyectofinalweb\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\laragon\www\proyectofinalweb\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ev...', 0)
#1 C:\laragon\www\proyectofinalweb\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ev...')
#2 C:\laragon\www\proyectofinalweb\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ev...')
#3 C:\laragon\www\proyectofinalweb\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ev...', Array, false)
#4 C:\laragon\www\proyectofinalweb\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\proyectofinalweb\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\proyectofinalweb\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\proyectofinalweb\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\proyectofinalweb\app\Controllers\EventoController.php(39): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\proyectofinalweb\system\CodeIgniter.php(933): App\Controllers\EventoController->store()
#10 C:\laragon\www\proyectofinalweb\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EventoController))
#11 C:\laragon\www\proyectofinalweb\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\proyectofinalweb\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\proyectofinalweb\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\proyectofinalweb\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-11-27 01:36:24 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
[Method: POST, Route: store]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (:nombreEvento:, :lugar:, :nombreOrganizador:, :cantInvitados:, :fecha:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\EventoController.php(39): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EventoController->store()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EventoController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-27 01:36:24 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (\'cumplea単os\', \'Mi casa\', \'cumplea単os\', \'50\', \'2024-06-17\', \'2024-11-27 01:36:24\', \'2024-11-27 01:36:24\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (\'cumplea単os\', \'Mi casa\', \'cumplea単os\', \'50\', \'2024-06-17\', \'2024-11-27 01:36:24\', \'2024-11-27 01:36:24\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (:nombreEvento:, :lugar:, :nombreOrganizador:, :cantInvitados:, :fecha:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\EventoController.php(39): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EventoController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EventoController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-27 01:36:24 --> [Caused by] mysqli_sql_exception: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (\'cumplea単os\', \'Mi casa\', \'cumplea単os\', \'50\', \'2024-06-17\', \'2024-11-27 01:36:24\', \'2024-11-27 01:36:24\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (\'cumplea単os\', \'Mi casa\', \'cumplea単os\', \'50\', \'2024-06-17\', \'2024-11-27 01:36:24\', \'2024-11-27 01:36:24\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (\'cumplea単os\', \'Mi casa\', \'cumplea単os\', \'50\', \'2024-06-17\', \'2024-11-27 01:36:24\', \'2024-11-27 01:36:24\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (:nombreEvento:, :lugar:, :nombreOrganizador:, :cantInvitados:, :fecha:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\EventoController.php(39): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EventoController->store()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EventoController))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-11-27 01:37:14 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\laragon\www\proyectofinalweb\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\laragon\www\proyectofinalweb\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ev...', 0)
#1 C:\laragon\www\proyectofinalweb\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ev...')
#2 C:\laragon\www\proyectofinalweb\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ev...')
#3 C:\laragon\www\proyectofinalweb\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ev...', Array, false)
#4 C:\laragon\www\proyectofinalweb\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\proyectofinalweb\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\proyectofinalweb\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\proyectofinalweb\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\proyectofinalweb\app\Controllers\EventoController.php(39): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\proyectofinalweb\system\CodeIgniter.php(933): App\Controllers\EventoController->store()
#10 C:\laragon\www\proyectofinalweb\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EventoController))
#11 C:\laragon\www\proyectofinalweb\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\proyectofinalweb\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\proyectofinalweb\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\proyectofinalweb\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-11-27 01:37:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
[Method: POST, Route: store]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (:nombreEvento:, :lugar:, :nombreOrganizador:, :cantInvitados:, :fecha:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\EventoController.php(39): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EventoController->store()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EventoController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-27 01:37:14 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (\'cumplea単os\', \'Mi casa\', \'cumplea単os\', \'50\', \'2024-06-17\', \'2024-11-27 01:37:13\', \'2024-11-27 01:37:13\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (\'cumplea単os\', \'Mi casa\', \'cumplea単os\', \'50\', \'2024-06-17\', \'2024-11-27 01:37:13\', \'2024-11-27 01:37:13\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (:nombreEvento:, :lugar:, :nombreOrganizador:, :cantInvitados:, :fecha:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\EventoController.php(39): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EventoController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EventoController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-27 01:37:14 --> [Caused by] mysqli_sql_exception: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (\'cumplea単os\', \'Mi casa\', \'cumplea単os\', \'50\', \'2024-06-17\', \'2024-11-27 01:37:13\', \'2024-11-27 01:37:13\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (\'cumplea単os\', \'Mi casa\', \'cumplea単os\', \'50\', \'2024-06-17\', \'2024-11-27 01:37:13\', \'2024-11-27 01:37:13\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (\'cumplea単os\', \'Mi casa\', \'cumplea単os\', \'50\', \'2024-06-17\', \'2024-11-27 01:37:13\', \'2024-11-27 01:37:13\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `eventos` (`nombreEvento`, `lugar`, `nombreOrganizador`, `cantInvitados`, `fecha`, `created_at`, `updated_at`) VALUES (:nombreEvento:, :lugar:, :nombreOrganizador:, :cantInvitados:, :fecha:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\EventoController.php(39): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EventoController->store()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EventoController))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-11-27 01:59:52 --> ParseError: syntax error, unexpected token "foreach"
[Method: GET, Route: /]
in APPPATH\Views\eventos\index.php on line 38.
 1 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('eventos/index', [], true)
 3 APPPATH\Controllers\EventoController.php(22): view('eventos/index', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\EventoController->index()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\EventoController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
